### ISSUE TITLE
SA1009 not reported inside conditional directives

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1834]
I expected SA1009 to be reported in the following, but no spacing diagnostics were reported.

```csharp
class Program
{
    static int Main(string[] args)
    {
#if !(X || NOT )
        return 1;
#endif
    }
}
```

:memo: Discovered while working on #1831.

**Comment by sharwell** [comment]
Found an additional case for the following:

```csharp
/// <see cref="Method( )"/>
public void Method()
{
}
```
